### ISSUE TITLE
How to setup ?

### ISSUE THREAD

**Issue by ora19982022** [Miki-Hunter/-store-server-django#1]
can u pls add the steps to setup the whole app？ thx.

**Comment by Miki-Hunter** [comment]
Well, there is no different with other django project:  
1、Install the required package( in requirement.txt )
2、Set up your database ( I used Mysql, and the sql file is prepared, rename it before use)
3、python manage.py makemigrations
      python manage.py migrate
4、Finally, runserve it.

Well, you may do some api-test job.

**Comment by ora19982022** [comment]
Thanks for the quick reply, did u use Pycharm to develop? or other tools? I plan to setup your dev environment, thanks.

The below is the list for the dev, correct? any specific version?
1. MySQL
2. Python
3. Django+RDF
4. Pycharm 

anything else? thanks.

**Comment by Miki-Hunter** [comment]
That`s right,  MySQL+Pycharm，and the package in requirement.txt,  "pip install -r requirement.txt" maybe。

Honestly, I only studied DRF for one week, though I finished this, it is definitely not the best way，"Stupid way to complete the task" ， we can learn together。

**Comment by ora19982022** [comment]
Looks like I cannot reset migration since there are data in tables.
run below command will cause errors:
python manage.py migrate
can u pls let me know the application name? so that I can reset the migrations? thanks.

can u pls let me know your wechat id and email address as well? my wechat id is YaoYuan6663, thanks.

**Comment by Miki-Hunter** [comment]
There may be many small problems that occur in this situation，I mean, every computer may have problems, and it is often different，so I may not be able to help you solve yours.
My email address: mikihunter11235@gmail.com, my QQ id :1970665218,  I hardly use WeChat.

**Comment by ora19982022** [comment]
Wiki您好，

已经可以运行了，这个应用只是电商平台的接口对么？不包括全部电商平台后台对吧？谢谢！

Regards,

Ora1998

莫让晗晗 ***@***.***> 于2022年8月8日周一 21:10写道：

**Comment by Miki-Hunter** [comment]
Only work for this project https://github.com/Miki-Hunter/vue-store-1.0

**Comment by ora19982022** [comment]
您好，不知道我是否没搞清楚。这个项目只是接口实现对么？不包括mall的全部后台管理？

谢谢！初学，所有有很多不太懂的地方。可否截几个屏我看下？

另外，我看了路由表，这些是所有的路由表，看起来都是接口。没看见有全部电商平台的后台路由，是不是我搞错了？谢谢！

urlpatterns = [
path('users/', include(router_users.urls)),
path('resources/', include(router_resources.urls)),
path('user/shoppingCart/', include(router_shoppingCart.urls)),
path('user/collect/', include(router_collect.urls)),
path('user/order/', include(router_order.urls)),
path('product/', include(router_product.urls)),
]

莫让晗晗 ***@***.***> 于2022年8月9日周二 09:17写道：

**Comment by Miki-Hunter** [comment]
我不知道是不是中英文翻译之后变了意思，“后端” 、 “后台” 是两个不一样的东西。

你现在clone的项目（store-server），是我为 https://github.com/Miki-Hunter/vue-store-1.0 （一个前端项目，叫它 toUser） 做的API。

有了这个（store-server），你就可以在你的电脑运行 toUser  。 

而至于你说的 “ 后台管理 ”， 也是一个前端的项目( 叫它 toAdmin )，toAdmin 和 toUser 可以共用DRF提供的API。

toAdmin 我正在做。

**Comment by ora19982022** [comment]
这就对了，看上去全部都是接口，好的，谢谢。我会再尝试着SETUP一下https://github.com/Miki-Hunter/vue-store-1.0（前端toUser），看能否跑起来。谢谢！
